OrbitQt/servicedeploymanager.cpp

```diff
@@ -29,6 +30,20 @@ static const std::chrono::milliseconds kSshWatchdogInterval(1000);
 
 namespace OrbitQt {
 
+namespace {
+
+void PrintAsOrbitService(const std::string& buffer) {
+  std::vector<std::string_view> lines = absl::StrSplit(buffer, '\n');
+  for (const auto& line : lines) {
+    if (!line.empty()) {
+      PLATFORM_LOG(
+          absl::StrFormat("[%28s] %s\n", "OrbitService", line).c_str());
```

Comment:
I would prefer to merge the two literals. No need to pay for the formatting at runtime.

Reply:
You mean this: 
`absl::StrFormat("[                OrbitService] %s\n", line).c_str()`? Will do it

Reply:
TIL: Github processes whitespaces in `code` comments. 
There is an appropriate amount of whitespaces between `[` and `OrbitService]`

Reply:
done

Reply:
Exactly that. Thanks.